repos/system_upgrade/el7toel8/actors/removeoldpammodulescheck/actor.py

```diff
@@ -44,7 +43,6 @@ class RemoveOldPAMModulesCheck(Actor):
                     key='confirm',
                     label='Disable pam_krb5 module in PAM configuration? '
                           'If no, the upgrade process will be interrupted.',
-                    default=False,
```

Comment:
Could you please elaborate on removing these "defaults"?

Reply:
There should be no defaults in the dialogs that require serious user decision. If we could have a default here - why have a dialog asking the user to take responsibility?

Reply:
Ok, fair enough. In the past there was a discussion that we may have something like "--default" to continue with upgrade with suggested choices but the idea was most probably completely abandoned...

Reply:
I thought that default will not be used by default,but only when user applies specific option like `--use-default`. Anyway, still it could leat into the situation that people will not read it at all, so removal of default is probably really better from that point.

Reply:
Resolving for now as per discussion with original actor writer having a default=True is a no go here and having default=False won't let upgrade to pass.